java-sdk/radar-schemas-tools/src/main/java/org/radarcns/schema/registration/SchemaRegistry.java

```diff
@@ -66,11 +68,23 @@
     public SchemaRegistry(String baseUrl) throws MalformedURLException {
         ServerConfig config = new ServerConfig(baseUrl);
         config.setUnsafe(true);
-        this.schemaClient = new SchemaRetriever(config, 10);
         this.httpClient = RestClient.global()
             .timeout(10, TimeUnit.SECONDS)
             .server(config)
             .build();
+        this.schemaClient = new SchemaRetriever(this.httpClient);
+    }
+
+    public SchemaRegistry(String baseUrl, String apiKey, String apiSecret)
+            throws MalformedURLException {
+        ServerConfig config = new ServerConfig(baseUrl);
+        config.setUnsafe(true);
```

Comment:
This can be set to false, any schema registry that uses an API key should use a valid SSL certificate

Reply:
Yes. I had the exact question. Thanks for confirming